data/infra/resources/bash.yaml

```diff
@@ -99,74 +169,126 @@ properties_list:
   ruby_type: String, Integer
   required: false
   description_list:
-  - markdown: 'The user name or user ID that should be changed before running a
+  - markdown: The user name of the user identity with which to launch the new process.
+      The user name may optionally be specified with a domain, i.e. `domain\user`
+      or `user@my.dns.domain.com` via Universal Principal Name (UPN)format. It can
+      also be specified without a domain simply as user if the domain is instead specified
+      using the domain property. On Windows only, if this property is specified, the
+      password property must be specified.
+examples: |
+  **Compile an application**
 
-      command.'
-- property: umask
-  ruby_type: String, Integer
-  required: false
-  description_list:
-  - markdown: The file mode creation mask, or umask.
-resources_common_properties: true
-resources_common_notification: true
-resources_common_guards: true
```

Comment:
These were all undone on the resource updates. Is that correct?

Reply:
These were removed as duplicates. They are also listed at the top of the document on lines 2 - 5